src/main/resources/static/js/MainWindow.js

```diff
@@ -1,670 +1,769 @@
+/*global optimizeforsource, mappingData, currentDelimiter, currentHeaderLines, preview, addRow, delLastRow, resetMapping, showstep1, isExcel, getLocations, initDatastream, loadMapping, returnRows, showmessagetag, getCurrentFileName, openaccordion, upload*/
+
+var scrollToBottom = false;
+
 /**
+ * 
  * shows the popup to create a new Thing
  */
 function showThingModal() {
-    var $modal = $("#indexfooter").find("button:eq(0)");
-    $modal.attr("onclick", "createThing()");
-    $modal.html("Create Thing");
-    $modal.show();
-    modal("dialog", "thing.html", getLocations, "Create a Thing");
+	var $modal = $("#indexfooter").find("button:eq(0)");
+	$modal.attr("onclick", "createThing()");
+	$modal.html("Create Thing");
+	$modal.show();
+	modal("dialog", "thing.html", getLocations, "Create a Thing");
 }
 
 /**
  * opens the popup to create a new datastream
  */
 function showStreamModal() {
-    var $modal = $("#indexfooter").find("button:eq(0)");
-    $modal.attr("onclick", "createDS()");
-    $modal.html("Create Datastream");
-    $modal.show();
-    modal("dialog", "datastream.html", initDatastream, "Create a Datastream");
+	var $modal = $("#indexfooter").find("button:eq(0)");
+	$modal.attr("onclick", "createDS()");
+	$modal.html("Create Datastream");
+	$modal.show();
+	modal("dialog", "datastream.html", initDatastream, "Create a Datastream");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['initDatastream' is not defined.](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=4781409)